common/content/options.js

```diff
@@ -41,7 +41,7 @@ const Option = Class("Option", {
 
         // add no{option} variant of boolean {option} to this.names
         if (this.type == "boolean")
-            this.names = array([name, "no" + name] for (name in values(names))).flatten().__proto__;
+            this.names = util.Array.flatten(Array([name, "no" + name] for (name in values(names)))).__proto__;
```

Comment:
Uhm, I'm sorry.. I was busy and couldn't figure out quickly what was the issue here, so I postponed it. Maybe I'll give it a shot and if I do I will deliver a more robust PR :)